molecule/default/group_vars/airflow/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-airflow_fernet_key: xKy13nPFfDflJ0DYGVTwf_DEmbItfURHlEDxrt-bBQw=
\ No newline at end of file
+airflow_fernet_key: xKy13nPFfDflJ0DYGVTwf_DEmbItfURHlEDxrt-bBQw=
```

Comment:
If this is a secret it should not be in the role, you can use a placeholder here like `<AIRFLOW_FERNET_KEY>`, this value would be overrided later on the playbook.

Reply:
Nevermind, it's molecule testcase my bad.

Reply:
Is a default fernet key to role test works, in the playbook this key is in a vault.
If I put the value as a placeholder how can specify the key for travis?

Reply:
It's okay as it is, I haven't seen before that it is for a test scenario.